packages/heartwood-components/Gulpfile.js

```diff
@@ -1,31 +1,68 @@
-var gulp = require('gulp');
-var rename = require('gulp-rename');
-var sass = require('gulp-sass');
-var sassGlob = require('gulp-sass-glob');
-var concat = require('gulp-concat');
-var rename = require('gulp-rename');
-var uglify = require('gulp-uglify');
+var fs = require('fs-extra')
```

Comment:
Not that it matters much in this case, but while we're in here, let's switch to `const` for consistency

Reply:
Done; enforcing that with `.eslintrc`.